beacon-chain/blockchain/process_block.go

```diff
@@ -564,6 +564,8 @@ func (s *Service) isDataAvailable(ctx context.Context, root [32]byte, signed int
 		if len(sidecars) >= expected {
 			s.blobNotifiers.delete(root)
 			if err := kzg.IsDataAvailable(kzgCommitments, sidecars); err != nil {
+				log.WithField("root", fmt.Sprintf("%#x", root)).Warn("removing blob sidecars with invalid proofs")
```

Comment:
There are two possible errors, the length of commitments is not right or there is an invalid proof. It's safe to remove blobs anyway here.